src/store/Modules/Projects.ts

```diff
@@ -572,10 +575,11 @@ export default class Projects extends VuexModule {
       this.context.commit("updateTheme", themeData)
     } catch (err) {
       console.log(err)
+      this.context.commit("updateTheme", {})
```

Comment:
Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.